cime_config/config_grids.xml

```diff
@@ -4870,6 +4931,35 @@
       <map name="GLC2OCN_ICE_RMAPNAME">cpl/gridmaps/mpas.gis1to10km/map_gis1to10km_to_EC30to60E2r2_aave.210304.nc</map>
     </gridmap>
 
+    <!-- ====================    -->
+    <!-- GLC: mpas.gis1to10kmR2  -->
+    <!-- ====================    -->
+      
+    <gridmap glc_grid="mpas.gis1to10kmR2" lnd_grid="r05">
+      <map name="LND2GLC_FMAPNAME">cpl/gridmaps/r05/map_r05_to_gis1to10r02_mono.230725.nc</map>
```

Comment:
Everywhere above this grid has the suffix `R2` but here it has `r02`.  @jonbob , is that discrepancy ok?

Reply:
Good catch. I don't have an answer for that at the moment (will wait for Jon's response).

Reply:
They're sort of intermixed but it's defined on line 2915 as R2 -- so "glc_grid" needs to match. The mapping file names don't need to, though it may create some confusion. I'll need to regenerate a lot of these mapping files to match the v3 naming and algorithm convention, so we can decide as I do that